src/Nethermind/Nethermind.Serialization.Rlp/ReceiptStorageDecoder.cs

```diff
@@ -23,6 +23,14 @@ namespace Nethermind.Serialization.Rlp
 {
     public class ReceiptStorageDecoder : IRlpDecoder<TxReceipt>
     {
```

Comment:
Is eth63 GetReceipts served correctly - not sure if this is properly tested

Reply:
This is special decoder only for storage. There is separate ReceiptMessageDecoder for network communications